pkg/types/harness_simple_telemetry.go

```diff
@@ -94,6 +95,14 @@ type SimpleAMSIFields struct {
 	AppName     string `json:"app_name,omitempty"`
 }
 
+type SimpleRegFields struct {
+	EventType string `json:"event_type,omitempty"` // event_type: "SETVALUEKEY", "DELETEKEY", ...
+	Pid       int64  `json:"pid,omitempty"`
```

Comment:
no `hive` field?

Reply:
Yeah I didn't find any hive field in the bundles generated by Taegis windows agent.